tfe.go

```diff
@@ -639,22 +639,22 @@ func unmarshalResponse(responseBody io.Reader, model interface{}) error {
 	items := dst.FieldByName("Items")
 	pagination := dst.FieldByName("Pagination")
 
-	// Unmarshal a single value if v does not contain the
+	// Unmarshal a single value if model does not contain the
 	// Items and Pagination struct fields.
 	if !items.IsValid() || !pagination.IsValid() {
 		return jsonapi.UnmarshalPayload(responseBody, model)
 	}
 
-	// Return an error if v.Items is not a slice.
+	// Return an error if model.Items is not a slice.
 	if items.Type().Kind() != reflect.Slice {
-		return fmt.Errorf("v.Items must be a slice")
+		return fmt.Errorf("model field \"Items\" must be a slice")
```

Comment:
Can you change this to create a static error with errors.New. No need to use fmt.Errorf because we are not formatting anything. Weird that the golangci-lint didn't catch this, but maybe we have not enable one of those lint libraries that looks after unneeded formatting.
Let's have that errors.New as part of the errors.go file

Reply:
Oh, also, no need to escape quotes. Go uses `` back-ticks to allow for quotes inside of a string. So this should be errors.New(`Model field "Items" must be a slice`)

Reply:
Ironically, github is formatting my errors.New message inside of the parenthesis because I put backticks in there 😆